state.go

```diff
@@ -15,123 +15,19 @@ package main
-//                                  |
-//                                  |   +-----------------------------+
-//                                  |   |                             |
-//                                  v   v                             |
-//
-//                             update fetch ------------------> retry update
-//
-//                                  |                                 ^
-//                                  | (update fetched)                |
-//                                  v                                 |
-//                                                                    |
-//                            update install -------------------------+
-//
-//                                  |
-//                                  | (update installed,
-//                                  |  enabled)
-//                                  |
-//                                  v
-//
-//                                reboot
-//
-//                                  |
-//                                  v
-//
-//                                final (daemon exit)
-//
-// Errors and their context are captured in Error states. Non-update states
-// transition to an ErrorState, while update related states (fetch, install,
-// commit) transition to UpdateErrorState that captures additional update
-// context information. Error states implement IsFatal() method to check whether
-// the cause is fatal or not.
-//
-//        +------------------> init <-----------------------+
-//        |                                                 |
-//        |                      |                          |
-//        |                      |                          |
-//        |                      |                          |
-//        |                      v                          |
-//                                             (bootstrap)  |
-//   error state <--------- non-update states  (authorized) |
-//                                             (* wait)     |
-//        |                       ^            (check)      |
-//        |                       |                         |
-//        |                       |                         |
-//        |                       |                         |
-//        |      (fetch  )        v                         |
-//        |      (install)
-//        |      (enable )  update states ---------> update error state
-//        |      (verify )
-//        |      (commit )        |                         |
-//        |      (report )        |                         |
-//        |      (reboot )        |                         |
-//        |                       |                         |
-//        |                       v                         |
-//        |                                                 |
-//        +-------------------> final <---------------------+
-//                           (daemon exit)
-//
-
```

Comment:
Outdated documentation is worst than no documentation. Thanks for cleaning this up ;)